schema/crdb/inventory-omicron-sled-config/up16.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE omicron.public.inv_sled_agent
+    ADD CONSTRAINT IF NOT EXISTS
+    reconciler_status_duration_present_unless_not_yet_run CHECK (
+        (reconciler_status_kind = 'not-yet-run'
+            AND reconciler_status_duration_secs IS NULL)
+        OR
+        (reconciler_status_kind != 'not-yet-run'
+            AND reconciler_status_duration_secs IS NOT NULL)
+    );
```

Comment:
Can this constraint be combined with the one in `up15`?

Reply:
Yep, done :+1: